packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -146,7 +146,9 @@ export class TdsDropdown {
     if (this.filter) {
       this.inputElement.value = this.getValue();
     } else {
-      this.inputElement.value = selection.length > 0 ? selection[0].label : '';
+      if (this.inputElement) {
+        this.inputElement.value = selection.length > 0 ? selection[0].label : '';
+      }
```

Comment:
As we talked, this line of code seems to be skipped. I tried putting a console.log in condition, but it never comes up.
We need to find another way to resolve it.